components/styled/header.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components/native';
+
+const Header = styled.Text`
+  font-family: 'Montserrat-Regular';
+`;
+
+export default Header;
```

Comment:
Usually you can do this in each individual file unless it’s being used in multiple places, but in this case I don’t think Header needs its own component and file. Please remove this and just put it in the files you’re using